SparkPost/2web2ui
# EO-696 Properly namespace global reports css

### What Changed
- Global CSS classes in the summary and engagement charts are now nested under class wrappers.
- CSS should no longer leak between summary, engagement, and signals.
- No actual changes to signals charts, report charts were leaking CSS.

### How To Test
- [Point your local upstreams to staging or production](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams)
- Log into appteam or account 108 for data

(1) Health Score Reference Lines
- Visit V2 health score dashboard or details page & hard refresh (Reference lines should have color)
- Visit summary report and return to health score  (Reference lines should **still** have color)

(2) Engagement Report
- Visit engagement report & hard refresh (axis lines should not be black)
- Edit: Wasn't part of the AC, but I noticed while fixing the first issue. I added some CSS here to make it more consistent with the summary chart

### To Do
